agent/dehydrate.js

```diff
@@ -10,6 +10,8 @@
  */
 'use strict';
 
+const LEVEL_THRESHOLD = 1;
```

Comment:
Maybe leave a comment here about perf implications? e.g. "This used to be set to 2 but it caused too much overhead in large apps and we decided to set it to 1 instead"